docs/building/windows-core.md

```diff
@@ -77,3 +77,13 @@ You can run our cross-platform Pester tests with `Start-PSPester`.
 ## Building in Visual Studio
 
 We currently have the issue [#3400](https://github.com/PowerShell/PowerShell/issues/3400) tracking this task.
+
+## Building an installer
```

Comment:
We use `Start-PSPackage` function. See https://github.com/PowerShell/PowerShell/blob/cb048ea9f4f7ccc3612141ee76f7937b9ca1c823/docs/maintainers/releasing.md

Reply:
`Start-PSPackage` is not very developer friendly because it has ridiculous requirements such as that the checkout must be at the root of the filesystem and that PowereShell is compiled in a very specific way. The  purpose of this PR is to make it easier to build MSIs locally when changing WiX code.

Reply:
`Start-PSPackage` also does some other checks, but that's more intended for creating a release package and I can understand the scenario for a developer testing their wix changes.  However, it's actually pretty easy to have `PowerShell` repo as your root using either `New-PSDrive` or `Subst`.

Reply:
I think we should support one entry point - it is easily to support (in docs too).
Also it will be best to have the same code (csproj?) to build on local and on GitHub.